.hygen/generate/relational-resource/service.spec.ejs.t

```diff
@@ -1,8 +1,10 @@
 ---
-to: src/<%= h.inflection.transform(name, ['pluralize', 'underscore', 'dasherize']) %>/<%= h.inflection.transform(name, ['pluralize', 'underscore', 'dasherize']) %>.service.spec.ts
+to: "<%= isAddTestCase ? `src/${h.inflection.transform(name, ['pluralize', 'underscore', 'dasherize'])}/${h.inflection.transform(name, ['pluralize', 'underscore', 'dasherize'])}.service.spec.ts` : null %>"
 ---
 import { Test, TestingModule } from '@nestjs/testing';
-import { paginationOptions, mock<%= name %>,mockCreate<%= name %>Dto, mockUpdate<%= name %>Dto } from './__mock__/<%= h.inflection.transform(name, ['underscore', 'dasherize']) %>.mock';
+<% if (functionalities.includes('findAll')) { %>
+import { paginationOptions, mock<%= name %>, mockCreate<%= name %>Dto, mockUpdate<%= name %>Dto } from './__mock__/<%= h.inflection.transform(name, ['underscore', 'dasherize']) %>.mock';
+<% } %>
 import { <%= h.inflection.transform(name, ['pluralize']) %>Service } from './<%= h.inflection.transform(name, ['pluralize', 'underscore', 'dasherize']) %>.service';
 import { <%= name %>Repository } from './infrastructure/persistence/<%= h.inflection.transform(name, ['underscore', 'dasherize']) %>.repository';
```

Comment:
I think this import should be like this :
`import { <%= name %>Repository } from './infrastructure/persistence/<%= h.inflection.transform(name, ['underscore', 'dasherize']) %>.abstract.repository';`

Can you please verify on your end?

Reply:
done updated